packages/gdl-frontend/lib/analytics.js

```diff
@@ -24,12 +24,13 @@ export function logPageView() {
 }
 
 // Define all allowed categories here, so we are able to group all datapoints correctly
-type Category = 'Books' | 'User' | 'Navigation' | 'PWA';
+export type Category = 'Books' | 'Games' | 'User' | 'Navigation' | 'PWA';
 type Action =
   // User
   | 'Login'
   // Books
   | 'Read'
+  | 'Play'
```

Comment:
La til en ny kategori for å logge event når brukeren trykker på link for å gå til game link. Usikker på om dette er nok eller det må gjøres no konfig google analytics.

Reply:
Tror ikke det må gjøres noe på Google Analytics